_posts/2023-04-28-date-time-testing.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+authors: [maarten_casteels]
+title: 'Date and Time Testing'
+image: /img/2023-04-28-date-time-testing/blog-post-overlay.webp
+tags: [testing]
+category: Testing
+comments: false
+---
+
+Working with dates and times can be challenging for developers, especially regarding testing. 
+When testing code that involves the current date or time, it takes time to ensure that the results are correct and consistent. 
+Luckily, there are several ways to solve this problem.
+
+One approach is to use a fixed date and time in your tests. 
+This ensures that your code produces consistent results, regardless of the actual date and time. 
+You can achieve this by mocking the `now()` method of the `LocalDateTime` class, which returns the current date and time.
+
+Here's an example of how to mock the `now()` method of the `LocalDateTime` class using Mockito and Java:
```

Comment:
Zou nog een stukje toevoegen dat je mockito inline hebt gebruikt ipv de mockito core. Om static methods te kunnen mocken.

Reply:
@bastijns-jeroen , I solved it, and added the dependency